app/notifiers/panopticon_notifier.rb

```diff
@@ -0,0 +1,14 @@
+require 'gds_api/panopticon'
+require 'ostruct'
+
+class PanopticonNotifier
+  def self.call
```

Comment:
Strongly agree, didn't see an immediate need but it's a good habit anyway.

I'll be keeping the class method and delegating to new like I've done below tho' ;).